varnish-http-purge.php

```diff
@@ -347,94 +380,131 @@ public function purgeNoID( $postId ) {
 	 */
 	public function purgePost( $postId ) {
 		
+		global $wp_version;
```

Comment:
Hah, that's a remnant from when I was spec'ing out how I wanted to handle 'what if WP changes the JSON versions!!!'